setup.py

```diff
@@ -132,6 +132,7 @@ def main():
           ext_modules=cythonize(["hummingbot/**/*.pyx"], language="c++", language_level=3, nthreads=cpu_count),
           include_dirs=[
               np.get_include(),
+              "D:\\GITHub\\hummingbot\\"
```

Comment:
Remove this line.

What was broken to cause you to add this?

Reply:
Sorry, this was from when I was trying to get hummingsim to work on windows, I will remove this.